boards/amd/versal2_rpu/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2025 Advanced Micro Devices, Inc.
+#
+
+if (EXISTS "${BOARD_DIR}/${BOARD}-qemu.dts")
+  set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+    COMMAND dtc -I dts -O dtb -q ${BOARD_DIR}/${BOARD}-qemu.dts -o ${PROJECT_BINARY_DIR}/${BOARD}-qemu.dtb
```

Comment:
should be using proper variable and with `find_package(Dtc 1.4.6)`, if required then should have required flag

Reply:
sure will fix